### PR TITLE
feat:CO-60 : apply button changes as per new design

### PR DESCRIPTION
**Pull request by arjun-at-roam**
## PR Area

- [x] Frontend

## Type of change

_What kind of change does this PR introduce?_

- [x] Feature

## Description

Design:
image

Before:
image
image

After:
image
image

* Added a new color to the palette (color-3) that will only be used for the "Learn More" button and had to nudge other color-3, color-4, color-5 and color-6 by one
* Added a new bg color to the palette (bg-2) for dialog background
* Added 3 new props to ApplyCard module
  * buttonPrimary
  * buttonColor
  * buttonLabel

## Security & risks

- [ ] Adds new third party libraries
- [ ] Adds new `.env` variables
- [ ] Integrates with Third-party apis (dependency must be resilient)
- [ ] Has specific [OWASP security concerns](https://owasp.org/www-project-top-ten/) - list them: (also see [Fixing OWASP Top 10](https://nodegoat.herokuapp.com/tutorial))

## Pre-publish checklist

- [x] I have assigned myself to the ticket
- [x] I have performed a self-review of my code
- [x] I have performed relevant browser testing
- [ ] I have performed keyboard testing of new or modified interactive components

## Reviewer checklist

- [ ] I have run the code (if applicable)
- [ ] I have reviewed and run any tests (if applicable)

### CODE REVIEW

File: src/providers/Theme.vue [hunk 1]
```diff
@@ -66,6 +70,7 @@ export default defineComponent({
     --color-1: #ffffff;
     --color-1-contrast: #ffffff;
     --color-2: #573a90; // Web / Tile / Button / Secondary
+    --color-3: #ffffff; // Main Widget Button
     --bg-1: linear-gradient(
       90deg,
       #b83990 18.24%,
```
Comment (arjun-at-roam) [inline src/providers/Theme.vue:76]: --color-3 will be used only for main widget button doing forward as it doesn't seem to have a relation to the other button colors. Checkout design 👆 
image

**Review by ghost — APPROVED** [review]
Nice job! I love seeing it all together and the theming working this well 🚀

File: src/providers/Theme.vue (hunk 1 above)
Comment (ghost) [reply src/providers/Theme.vue:76]: If the number of theme colours grow any larger it's a good idea to add even more comments per colour for where it's used 🙏

**Review by antoninroam — APPROVED** [review]

**Review by jpyepez-roam — APPROVED** [review]
LGTM! 🚀

**Pull request merged**